InfiniteRecharge/src/main/java/frc/robot/config/ConfigChooser.java

```diff
@@ -19,7 +19,7 @@
     //
     // Change this type to pick config
     //
-    private static ConfigType configType = ConfigType.Main;
+    private static ConfigType configType = ConfigType.Junior;

```

Comment:
We shouldn't commit this. It should be changed locally if needed, but the master branch should always point to Main.